instrumentation/rabbitmq-2.7/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rabbitmq/RabbitTracer.java

```diff
@@ -76,7 +76,8 @@ public Context startGetSpan(
     NetPeerUtils.INSTANCE.setNetPeer(spanBuilder, connection.getAddress(), connection.getPort());
     onGet(spanBuilder, queue);
 
-    return Context.current().with(spanBuilder.startSpan());
+    // TODO: withClientSpan()?
+    return parentContext.with(spanBuilder.startSpan());
```

Comment:
I think so for consistency (at least until https://github.com/open-telemetry/opentelemetry-specification/issues/1360)

Reply:
We'll have to revisit all CLIENT instrumentations anyway once the convention changes - until then I think that we should be consistent and keep using `withClientSpan()` and `shouldStartSpan()`